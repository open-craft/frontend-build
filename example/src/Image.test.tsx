import renderer from 'react-test-renderer';
import Image from './Image';

describe('Basic test', () => {
  it('should render', () => {
    const Component = <Image src="foo" alt="alt" />;
    const tree = renderer.create(Component);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
