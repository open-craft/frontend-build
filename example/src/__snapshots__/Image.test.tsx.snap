// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic test should render 1`] = `
<img
  alt="alt"
  src="foo"
/>
`;
